Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Design/Common/MetadataRegistrationBase.cs

```diff
@@ -0,0 +1,252 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Windows.Design.Metadata;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Xml.Linq;
+
+namespace Microsoft.Toolkit.Uwp.Design.Common
+{
+	public class MetadataRegistrationBase
+	{
+        private AttributeTable masterMetadataTable;
+
+        internal MetadataRegistrationBase() { }
+
+        /// <summary>
+		/// Build design time metadata attribute table.
+		/// </summary>
+		/// <returns>Custom attribute table.</returns>
+		protected virtual AttributeTable BuildAttributeTable()
+		{
+			AttributeTableBuilder builder = new AttributeTableBuilder();
+
+            AddDescriptions(builder);
+            AddAttributes(builder);
+            AddTables(builder, this);
+            masterMetadataTable = builder.CreateTable();
+			return masterMetadataTable;
+		}
```

Comment:
I'm seeing weird spacing/formatting in lots of files from this PR, I'm wondering if you have some mixed tabs/spaces, which might explain why the GitHub website is rendering text with the wrong alignment? 🤔

For reference, here's what I'm seeing from here:

![image](https://user-images.githubusercontent.com/10199417/90509924-653ca180-e15a-11ea-8358-cd0a5709a3aa.png)

Not sure if you have it already, but I recommend this neat plugin for VS: [Fix Mixed Tabs](https://marketplace.visualstudio.com/items?itemName=VisualStudioPlatformTeam.FixMixedTabs). It automatically detects them whenever you open a file and it has a handy "untabify" option that just converts all tabs to spaces automatically. Hope it helps! 😊

Reply:
The 3 files inside this folder are exactly the same for all 3 Design projects, so I literally just copied it. This was there before. I'll fix it on all of them.